common/v2/features/BroadcastTx/BroadcastTx.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { Component } from 'react';
+import { RouteComponentProps } from 'react-router';
+import { connect } from 'react-redux';
+import { toBuffer, bufferToHex } from 'ethereumjs-util';
+import EthTx from 'ethereumjs-tx';
+import styled from 'styled-components';
+
+import translate, { translateRaw } from 'translations';
+import { computeIndexingHash, getTransactionFields, makeTransaction } from 'libs/transaction';
+import { AppState } from 'features/reducers';
+import * as selectors from 'features/selectors';
+import { transactionSignActions } from 'features/transaction';
+import { SendButton } from 'components/SendButton';
+
+import { ExtendedContentPanel, InputField, QRCode, CodeBlock } from 'v2/components';
+import { Button, Identicon } from '@mycrypto/ui';
+
+interface StateProps {
+  stateTransaction: AppState['transaction']['sign']['local']['signedTransaction'];
+}
+interface DispatchProps {
+  signLocalTransactionSucceeded: transactionSignActions.TSignLocalTransactionSucceeded;
+  signTransactionFailed: transactionSignActions.TSignTransactionFailed;
+}
+interface State {
+  userInput: string;
+  inputError: string;
+}
+const INITIAL_STATE: State = { userInput: '', inputError: '' };
+
+type Props = DispatchProps & StateProps & RouteComponentProps<{}>;
+
+const getStringifiedTx = (serializedTx: Buffer) =>
+  JSON.stringify(getTransactionFields(makeTransaction(serializedTx)), null, 2);
+
+const InputWrapper = styled.div`
+  margin-top: 30px;
+  display: flex;
+  align-items: center;
+`;
+
+const PlaceholderButton = styled(Button)`
+  opacity: 0.4;
+  margin-top: 20px;
+  cursor: default;
+`;
+
+const CodeBlockWrapper = styled.div`
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  max-width: 510px;
+  overflow-x: auto;
+`;
+
+const QRCodeWrapper = styled.div`
+   {
+    max-width: 15rem;
+    margin: 1rem auto;
+    width: 100%;
+    text-align: center;
+  }
+`;
+
+const IdenticonIcon = styled(Identicon)`
+  margin-left: 12px;
+  margin-top: 8px;
+
+  img {
+    width: 48px;
+    height: 48px;
+    max-width: none;
+  }
+`;
+class BroadcastTx extends Component<Props> {
+  public state: State = INITIAL_STATE;
+
+  public render() {
+    const { userInput } = this.state;
+    const stateTransaction = userInput ? this.props.stateTransaction : '';
+
+    return (
+      <ExtendedContentPanel
+        heading={translateRaw('BROADCAST_TX_TITLE')}
+        description={translate('BROADCAST_TX_DESCRIPTION')}
+        centered={true}
+        maxWidth="650px"
```

Comment:
Can we add this value in a shared const in `v2/config/constants` so we can start to apply all our panels?

Reply:
Moved to constants, unfortunately we can't just set this for all panels, since the designs are not consistent in this aspect.